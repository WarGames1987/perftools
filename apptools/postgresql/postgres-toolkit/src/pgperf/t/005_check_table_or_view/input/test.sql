-- table
SELECT pgperf._check_table_or_view('pg_database');
-- view
SELECT pgperf._check_table_or_view('pg_stat_activity');

SELECT pgperf._check_table_or_view('nosuchtableorview');
