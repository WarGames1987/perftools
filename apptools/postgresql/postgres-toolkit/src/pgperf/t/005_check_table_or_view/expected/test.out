SELECT pgperf._check_table_or_view('pg_database');
 _check_table_or_view 
----------------------
 t
(1 row)

SELECT pgperf._check_table_or_view('pg_stat_activity');
 _check_table_or_view 
----------------------
 t
(1 row)

SELECT pgperf._check_table_or_view('nosuchtableorview');
 _check_table_or_view 
----------------------
 f
(1 row)

